inc/admin/options-page.php

```diff
@@ -187,6 +211,29 @@ public function field_text( $args ) {
 	}
 
 	/**
+	 * Output a checkbox.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param  array  $args  The array of field args.
+	 */
+	public function field_checkbox( $args ) {
+
+		$option_id          = 'alcatraz-options-' . str_replace( '_', '-', $args['id'] );
+		$option_key         = 'alcatraz_options[' . $args['id'] . ']';
+		$option_value       = alcatraz_true_or_false( $this->options[ $args['id'] ] );
```

Comment:
There is an issue here. If the option key doesn't yet exist, it won't be set when this is first called, and you'll get a PHP Notice:

```
PHP Notice:  Undefined index: show_social_urls in /srv/alcatraz/wp-content/themes/alcatraz/inc/admin/options-page.php on line 224
```

This should be: 

```
$option_value = ( ! empty( $this->options[ $args['id'] ] ) ) ? alcatraz_true_or_false( $this->options[ $args['id'] ] ) : '';
```